test/integration/gc_test.go

```diff
@@ -20,6 +20,7 @@ package integration
 
 import (
 	"context"
+	"github.com/yorkie-team/yorkie/pkg/document/time"
```

Comment:
Could you put this import to the below group?

https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering